packages/trader/src/sass/app/modules/trading-mobile.scss

```diff
@@ -185,7 +185,19 @@
                 }
             }
         }
-        &__amount {
+
+        &__amount,
+        &__stake {
+            &-container {
+                margin: 0.4rem 0 -0.7rem;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+
+                .trade-container__turbos-trade-info {
+                    width: 240px;
```

Comment:
Use `rem`

Reply:
No problem, I've just tried to do as in code above, where for width and height % or px were used